stories/Welcome.stories.mdx

```diff
@@ -3,31 +3,53 @@ import './main.scss'
 
 <Meta title="Welcome" />
 
-# The OKP4 Design System
+# Welcome!
 
-## Objectives
+> The OKP4 Design System is a collection of clear guidelines, components, and resources that was created to aid in the design and development of our
+Web3.0 digital products in the [OKP4](https://okp4.network) Ecosystem, and more generally in the [Cosmos](https://cosmos.network) Ecosystem.
 
-The OKP4 Design System is a collection of clear guidelines, components, and resources that was created to aid in the design and development of our
-digital products - primarily our web apps or websites - for our customers.
+<div style={{ textAlign: 'center', padding: '10px 0 10px' }}>
+  <a href="https:/github.com/okp4/ui" style={{ padding: '0 10px' }}>
+    <img src="https://img.shields.io/badge/github-%23121011.svg?style=for-the-badge&amp;logo=github&amp;logoColor=white" alt="GitHub" />
+  </a>
+  <a href="https://discord.gg/GHNZh4SaJ3" style={{ padding: '0 10px' }}>
+    <img src="https://img.shields.io/badge/Discord-%237289DA.svg?style=for-the-badge&amp;logo=discord&amp;logoColor=white" alt="Discord" />
+  </a>
+  <a href="?path=/docs/getting-started--page" style={{ padding: '0 10px' }}>
+    <img src="https://img.shields.io/badge/🚀%20Get-Started-lightgrey?style=for-the-badge" alt="Get started" />
+  </a>
+  <a href="https://opensource.org/licenses/BSD-3-Clause" style={{ padding: '0 10px' }}>
+    <img src="https://img.shields.io/badge/License-BSD_3--Clause-blue.svg?style=for-the-badge" alt="License" />
+  </a>
+</div>
 
-The OKP4 Design System serves as a reference or source of truth for the teams working on [OKP4](https://okp4.network) digital products, enabling to design and develop with speed,
-consistency, and quality.
+## What it is?
```

Comment:
Feels weird but maybe it's me:
```suggestion
## What it is
or
## What is it?
```

Reply:
Option 2 chosen. Thanks for catching this. 👍